### ISSUE TITLE
Change auto eject commands to a "Blocked" message

### ISSUE THREAD

**Issue by OliverDatGuy** [TotalFreedom/TotalFreedomMod#589]
Trolls always say something like "Do /rl to get a rocket launcher" and then about 30 billion people actually do it and get banned for it.

Just change it to a message like "This command is blocked."

**Comment by DDQ888** [comment]
I vouch for this.

**Comment by AlexFreedomMod** [comment]
Or it should not let it send at all and same should apply if a command has the words too, and then tell the sender ("Sorry, you are not allowed to tell people to do " + args[1] + "!")

The Code for the text:

sender.sendMessage("Sorry, You are not allowed to tell people to do " + args[1] + "!");

**Comment by OliverDatGuy** [comment]
There's easy ways around that. Just remove the auto eject...

**Comment by AlexFreedomMod** [comment]
Yeah, but if this was on totalfreedom no Op could remove it but the people that fork totalfreedommod will be able to fork it.

**Comment by OliverDatGuy** [comment]
...what are you talking about? I'm saying if it bans that then you could just say "Do slash R L" and other things. It's less hassle to just remove the auto eject and just block the command, no need to complicate it.

**Comment by AlexFreedomMod** [comment]
yeah or we could just block /rl, //mat, all the command people will do to get banned. (They are already banned but some people do it because they think its a rocket laucher) Instead of having it ban them it should tell them this command is blocked

**Comment by OliverDatGuy** [comment]
Oh my fucking god, have you even read what I said?

"Just change it to a message like "This command is blocked.""

**Comment by AlexFreedomMod** [comment]
@OliverDatGuy  i did read it i just forgot that is what you said.

**Comment by Wild1145** [comment]
Why not simple check for if a blocked command is mentioned in chat, and simple filter it out? Solves about 100 different problems at once.

**Comment by AlexFreedomMod** [comment]
Yeah, @Wild1145! That will solve 100 diffent problems at once.

**Comment by OliverDatGuy** [comment]
Not really, there are ways to bypass a filter, simply blocking the command and not auto ejecting will A: Stop the bad command being used. B: Stop trolls having any need to say it and C: Stop innocent people from getting kicked.
